types/registry.go

```diff
@@ -0,0 +1,66 @@
+package types
+
+import "time"
+
+// CompanyAddress represents a company address
+type CompanyAddress struct {
+	AdressLine1 string `json:"addressLine1"`
```

Comment:
Should we use `kebab-case` for JSON perhaps?  Seems more appropriate than `camelCase`.

I realize the NZ schema uses these exact key names, but since this file defines our own internal schema (albeit based on NZ version), we should strive to use more suitable and descriptive field names.  It would mean a bit more transformation work, but the names will more closely follow the field names in `nsp_data`

Reply:
Switched all field names to kebab case